### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by ermias5**

### CODE REVIEW

File: components/services/service-section.tsx
Comment (Melka1) [inline components/services/service-section.tsx:1]: wtf?

File: components/lib/firebase-crud/create-data.ts
Comment (Melka1) [inline components/lib/firebase-crud/create-data.ts:1]: correct everything! why did you include the data in the commit?

File: components/hero/Hero.tsx
Comment (Melka1) [inline components/hero/Hero.tsx:1]: Is it hard to include Error and loading?

**Review by Melka1 — CHANGES_REQUESTED** [review]

File: components/projects/project-card.tsx
Comment (Melka1) [inline components/projects/project-card.tsx:1]: What about adding a placeholder image if there is no image

File: components/lib/firebase-crud/create-data.ts
Comment (Melka1) [reply components/lib/firebase-crud/create-data.ts:1]: read the last comment?

**Review by Melka1 — CHANGES_REQUESTED** [review]

File: components/hero/Hero.tsx
Comment (Melka1) [reply components/hero/Hero.tsx:1]: line-33 remove the braces

File: components/lib/firebase-crud/post-data.ts
Comment (Melka1) [inline components/lib/firebase-crud/post-data.ts:1]: Amnesia alebh ende!!!!!!!!

File: components/projects/project-card.tsx
Comment (Melka1) [reply components/projects/project-card.tsx:1]: Remove the state, you don't need it

**Review by Melka1 — CHANGES_REQUESTED** [review]

File: app/dashboard/form-handler.tsx
Comment (Melka1) [inline app/dashboard/form-handler.tsx:1]: try to move the api request out of this file, just invoke it here, read about clean architecture.
and even if the createFireStoreData fails, it show the Alert that it was successful

**Review by Melka1 — CHANGES_REQUESTED** [review]

File: app/dashboard/form-handler.tsx
Comment (Melka1) [inline app/dashboard/form-handler.tsx:1]: convert this to a custom hook and return {formData, handleInputChange, handleFileChange, handleSubmit} and use it on the form-component, when you use the word "handle" make sure it is not a function that return a component.

File: components/theme-toggle/theme.tsx
Comment (Melka1) [inline components/theme-toggle/theme.tsx:1]: the name is problematic, theme should be enum not string "light" or "dark"... (think please!!)

**Review by Melka1 — CHANGES_REQUESTED** [review]

File: hooks/useFormHandler.ts
Comment (Melka1) [inline hooks/useFormHandler.ts:1]: handle submit api won't throw error so the try catch is useless

File: components/lib/firebase-crud/create-data.ts
Comment (Melka1) [inline components/lib/firebase-crud/create-data.ts:1]: work on your error handling

File: app/dashboard/form-service.ts
Comment (Melka1) [inline app/dashboard/form-service.ts:1]: line-40 this function doesn't throw error, therefore there isn't any to catch

**Review by Melka1 — CHANGES_REQUESTED** [review]

File: app/dashboard/form-service.ts
Comment (Melka1) [reply app/dashboard/form-service.ts:1]: nothing has changed

File: components/lib/firebase-crud/create-data.ts
Comment (Melka1) [reply components/lib/firebase-crud/create-data.ts:1]: nothing has changed

File: hooks/useFormHandler.ts
Comment (Melka1) [inline hooks/useFormHandler.ts:1]: Correct your error handling, this tells that it is always successful.

**Review by Melka1 — CHANGES_REQUESTED** [review]

File: components/projects/project-section.tsx
Comment (Melka1) [inline components/projects/project-section.tsx:1]: what is the use of changing the skills to lowercase?

**Review by Melka1 — CHANGES_REQUESTED** [review]

File: components/projects/project-section.tsx
Comment (ermias5) [reply components/projects/project-section.tsx:1]: to filter out projects with out case sensitive when data uploading

File: app/page.tsx
Comment (Melka1) [inline app/page.tsx:1]: Cant you use the footer style inside the the footer component

File: components/testimonials/testimonial-section.tsx
Comment (Melka1) [inline components/testimonials/testimonial-section.tsx:1]: remove line 37

**Review by Melka1 — CHANGES_REQUESTED** [review]

**Review by Melka1 — APPROVED** [review]

**Review by Melka1 — APPROVED** [review]

**Pull request merged**